doc/api/deprecations.md

```diff
@@ -660,6 +660,19 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+#### DEP00XX: HTTP 418 I'm a teapot Status
+
+Type: Documentation-only
+
+The HTTP `418 I'm a teapot` status code was defined in [RFC2324][] as an
```

Comment:
It's important to note that 418 is *not* an HTTP status code, merely an HTCPCP one. I'd just remove the "HTTP" wording.